package-versions/grouping-and-overrides/multiple-groups.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Multiple Groups
+parent: Grouping and Overrides
+grand_parent: Package Versions
+nav_order: 2
+---
+
+{% raw %}
+<script>(function(t,e,s,n){var o,a,c;t.SMCX=t.SMCX||[],e.getElementById(n)||(o=e.getElementsByTagName(s),a=o[o.length-1],c=e.createElement(s),c.type="text/javascript",c.async=!0,c.id=n,c.src="https://widget.surveymonkey.com/collect/website/js/tRaiETqnLgj758hTBazgdzqSORtCawzIPfKe_2BNdVN8xOmlvDpjVPJr1OGV1A0yQQ.js",a.parentNode.insertBefore(c,a))})(window,document,"script","smcx-sdk");</script>
+{% endraw %}
+
+# Multiple Named Groups
+
+To limit the number of groups that are necessary to manage your solution, multiple groups can be used to provide granular control over package versions in each project.
+
+For example, a bar can serve many drinks by combining a small number of bottles of alcohol and simple ingredients on the counter. Each group is like a bottle that when mixed with other groups can cover a wide range of scenarios with minimal duplication.
+
+## Examples
+
+#### Directory.Packages.props
+
+```xml
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <!-- When CentralManagementGroup is unspecified for a <PackageVersion>, the <PackageVersion> is part of the default global group. -->
+    <ItemGroup>
+        <PackageVersion Include="Serilog" Version="2.9.0" />
+        <PackageVersion Include="Newtonsoft.Json" Version="12.0.2" />
+        <PackageVersion Include="foo" Version="1.1.1" />
+    </ItemGroup>
+    <!-- Each <PackageVersion> within this <ItemGroup> belongs to the "A" group as specified in their CentralManagementGroup attributes. -->
+    <ItemGroup>
+        <PackageVersion Include="Newtonsoft.Json" Version="11.0.2" CentralManagementGroup="A" />
+        <PackageVersion Include="foo" Version="2.2.2" CentralManagementGroup="A" />
+        <PackageVersion Include="qux" Version="2.0.0" CentralManagementGroup="A" />
+        <PackageVersion Include="xyzzy" Version="2.0.0" CentralManagementGroup="A" />
+        <!-- bar has a dependency on foo 202.0.0 -->
+        <!-- bar has a dependency on qux 202.0.0 -->
+        <!-- bar has a dependency on xyzzy 202.0.0 -->
```

Comment:
These comments use versions that don't appear anywhere else. I don't understand what's being indicated here. Is it that we're explicitly _downgrading_ the version dependencies? Such versions as these appear in multiple places elsewhere in the docs.